Update users.js

Scholldier opened the pull request:
This resolves issue #3057 and sorts the display name column alphabetically.

Scholldier committed:
Update users.js

Scholldier committed:
Update users.js

Scholldier committed:
Update users.js

jniles reviewed (approved):
Excellent!  Looks good to me.  I think I may have to dig into the tests to make sure all is well, which I will do after church.  But great work!

jniles committed:
fix(users): better test hooks

The test hooks have been re-written to refer to users by their
display_name instead of indices in the user grid.

jniles committed:
Merge remote-tracking branch 'upstream/master' into Scholldier-issue-3057

jniles committed:
fix(test): update cashbox test

This commit updates the cashbox permission test to make sure users are
properly assigned.
